doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -454,6 +454,37 @@ Field              | Type       | Meaning
 `cpu_nano_cores`   | [`bigint`] | Approximate CPU usage, in billionths of a vCPU core.
 `memory_bytes`     | [`bigint`] | Approximate RAM usage, in bytes.
 
+### `mz_storage_source_statistics`
+
+The `mz_storage_source_statistics` table gives the last known storage-specific statistics
+for all processes of all extant storage replicas.
+
+At this time, we do not make any guarantees about the exactness or freshness of these numbers.
+
+Field                 | Type        | Meaning
+----------------------|-------------|--------
+`id`                 | [`text`]     | The ID of the storage object (source or sink).
+`worker_id`          | [`bigint`]   | An identifier of a process within a host.
+`snapshot_committed` | [`boolean`]  | Whether or not we have committed the initial snapshot for a source.
+`messages_received`  | [`bigint`]   | How many messages we have received from upstream.
+`messages_committed` | [`bigint`]   | How many messages we have committed into our storage layer.
+`bytes_received`     | [`bigint`]   | How many bytes worth of messages we have received from upstream.
+
+These values all require some amount of interpretation:
+
+- `messages_received` does not distinguish between inserts and deletes. Some upstream sources, like postgres,
+  produce deletion messages that we count the same as insertions.
+- `messages_committed` can be permanently lower or higher than `messages_received`, because:
+  - The storage layer can choose to consolidate messages within 1 second
```

Comment:
nit: one second happens to be the current (default) setting but it might change, so maybe just talk about "within a certain time interval"?

Reply:
will change this!